src/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import CommentInfo from '../CommentInfo/CommentInfo';
+import { Comment } from '../types';
+
+type Props = {
+  comments: Comment[] | null;
+};
+
+const CommentList:React.FC<Props> = ({ comments }) => {
+  return (
+    <div>
+      COMMENTS
+      {comments ? comments.map(e => <CommentInfo comment={e} />) : ''}
```

Comment:
use logical operator &&
also use explicit naming, not e (what is e?), but comment, post, etc

```suggestion
      {comments ? comments.map(comment => <CommentInfo comment={comment} />) : ''}
```

Reply:
also add key to CommentInfo